hack/release.sh

```diff
@@ -229,7 +229,7 @@ function push_manifest_driver() {
   do 
     osv=$(lcase "${OSVERSION}")
     BASEIMAGE=mcr.microsoft.com/windows/nanoserver:${OSVERSION}; 
-    full_version=$(docker manifest inspect "${BASEIMAGE}" | jq -r '.manifests[0].platform["os.version"]'); 
+    full_version=$(docker manifest inspect "${BASEIMAGE}" | grep '"os.version"' | sed 's/.*"os.version": "\(.*\)".*/\1/')
```

Comment:
do we need cut in the end to fetch exact os.version value ??

Reply:
yes, we get 

```
% docker manifest inspect mcr.microsoft.com/windows/nanoserver:1809 | grep '"os.version"'
            "os.version": "10.0.17763.6189"
```

I need to get version value

```
% docker manifest inspect mcr.microsoft.com/windows/nanoserver:1809 | grep '"os.version"' | grep '"os.version"' | sed 's/.*"os.version": "\(.*\)".*/\1/'
10.0.17763.6189
```